packages/tradershub/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { useMemo } from 'react';
 import { useHistory } from 'react-router-dom';
-import { useAuthorize, useJurisdictionStatus } from '@deriv/api';
+import { useActiveAccount, useJurisdictionStatus } from '@deriv/api';
+import { Provider } from '@deriv/library';
 import { Button, Text } from '@deriv/quill-design';
 import { TradingAccountCard } from '../../../../../components/TradingAccountCard';
 import { THooks } from '../../../../../types';
-import { MarketTypeDetails } from '../../../constants';
+import { CFDPlatforms, MarketTypeDetails } from '../../../constants';
+import { TradeModal } from '../../../modals/TradeModal';
 import { MT5AccountIcon } from '../MT5AccountIcon';
 
 const AddedMT5AccountsList = ({ account }: { account: THooks.MT5AccountsList }) => {
-    const { data: activeWallet } = useAuthorize();
+    const { data: activeAccount } = useActiveAccount();
```

Comment:
Lets use activeTradingAccount just to be extra safe 😸

Reply:
@thisyahlen-deriv done